R/read_process_req_file.R

```diff
@@ -54,7 +54,17 @@ identify_duplicate_reqs = function(content_df){
   }
 }
 
-remove_comments_from_req = function(content_df) {
+strip_comments = function(content){
+  #' @param content vector of characters
+  #' @return vector of chracters without comments or -r and trimmed whitespace
+  #' @details This was originally part of another function but was split out to be used
+  #' in a few different places.
+  content = gsub("\\-r.*|#.*", "", content)
```

Comment:
A potential edge case where removing `-r` could have an adverse effect on package detection.  Admittedly, it's very edge, but a possible solution could be to have a flag parameter for removing `-r` in addition to comments?

```r
> strip_comments("20-random::randomNumbers(n = 1, col = 1)")
[1] "20"
```